.circleci/config.yml

```diff
@@ -43,8 +43,6 @@ jobs:
       - image: << parameters.image-name >>
     steps:
       - checkout
-      - run:
-          command: apt-get install -y jq
```

Comment:
This was causing other CI to fail, since we use different images that require `sudo` or use `apt-get` instead.